test_data/default_account.py

```diff
@@ -0,0 +1,5 @@
+default_firstName="Tam"
+default_lastName="Doan"
+default_pwd="Test@123"
+changed_pwd_01="Test@1234"
+changed_pwd_02="Test@12345"
```

Comment:
should not hande this. Please handle like a dictionary
{firstname: Tam; lastname: Doan .....}

Reply:
I created file /test_data/default_data.yaml to handle the variables